editoast/src/infra_cache/mod.rs

```diff
@@ -795,6 +798,8 @@ impl InfraCache {
         let mut track_ranges = vec![];
         let mut used_switches = HashMap::new();
 
+        let mut ordered_route_elements = vec![];
```

Comment:
We don't need this. We can transform the `RoutePath` struct like this:

```rust
#[derive(Debug, Clone)]
pub struct RoutePath {
    pub track_ranges: Vec<DirectionalTrackRange>,
    pub switches_directions: Vec<(Identifier, Identifier)>, // <-- Transform the HashMap into a Vec to keep nodes order
}
```

Reply:
done